overlord/snapstate/snapstate.go

```diff
@@ -65,7 +65,6 @@ const (
 // control flags for "Configure()"
 const (
 	IgnoreHookError = 1 << iota
-	TrackHookError
```

Comment:
this is fine because they values were not serialized afaict, otherwise we would have needed to keep the value reserved